tests/Microsoft.Bot.Builder.Dialogs.Declarative.Tests/JsonLoadTests.cs

```diff
@@ -60,6 +65,12 @@ await BuildTestFlow(@"Root.dialog")
             .StartTestAsync();
         }
 
+        [TestMethod]
+        public void JsonDialogLoad_CycleDetectionWithNoCycleMode()
+        {
+            Assert.ThrowsException<Exception>(() => BuildNoCycleTestFlow(@"Root.dialog"));
```

Comment:
Don't we want to catch InvalidOperationException here? Even though the current assertion is true, a more specific assertion would be better.

```suggestion
            Assert.ThrowsException<InvalidOperationException>(() => BuildNoCycleTestFlow(@"Root.dialog"));
```